src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/20251001163656_EES6080_RenameExpiryToExpiresInPreviewTokensTable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Migrations
+{
```

Comment:
Minor one which you can ignore if you want - Lee refactored all existing migrations to use a file-scoped namespace and I usually use my IDE to refactor new ones straight after generating them.

So you could do the same here but it really doesn't matter. This is more to make you aware of it.